src/app/aliases/aliases.component.ts

```diff
@@ -18,8 +18,8 @@ export class AliasesComponent extends Base implements OnInit {
   organization$: Observable<Organization>;
   collection$: Observable<Collection>;
 
-  public type: string;
-  public alias: string;
+  public type: string | null;
```

Comment:
I would prefer `public type?: string`. Can't say exactly why, but it feels better to me.

@garyluu In the GitHub issue, you suggested this approach. Why do you like it?

Reply:
I think I only prefer it in interfaces (like ExtendedWorkflow.ts) where the property is optional because the compiler would complain when you try to create the object without that property.